helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -172,4 +172,13 @@ export class CheckChatStatus {
 				return;
 			});
 	}
+
+	private async getAnalyticsQueueTime(read: IRead, assoc: RocketChatAssociationRecord) {
+		const { queueTimeStart } = await retrievePersistentData(read, assoc);
+		if (queueTimeStart === null) {
+			throw new Error(ErrorLogs.QUEUE_TIME_NOT_SET);
```

Comment:
What happens if this error gets thrown?  Will we still send the ESCALATION_SUCCESSFUL event?  Basically, we do not want to kill the event if the timer fails, we just want to send an empty string.

Reply:
Fixed.